pkgs/development/tools/rust/cargo-pretty-test/default.nix

```
@@ -0,0 +1,21 @@
+{lib, fetchgit, stdenv, fetchFromGitHub, rustPlatform}:
```

Perhaps, these are unnecessary, and you could rewrite this to use fetchFromGitHub, since you're using a GitHub repo anyways?
```suggestion
{lib, fetchFromGitHub, rustPlatform}:
```